### PR TITLE
Stick to CUDA 11.8 in CI

### PR DESCRIPTION
**Pull request by ye-luo**
## Proposed changes
Unblock CI.

## What type(s) of changes does this code introduce?
- Testing changes (e.g. new unit/integration/performance tests)

### Does this introduce a breaking change?
- No

## What systems has this change been tested on?
Not tested. See CI results.

## Checklist
- Yes. This PR is up to date with current the current state of 'develop'

### CODE REVIEW

**Comment by ye-luo** [comment]
Test this please

**Comment by ye-luo** [comment]
Test this please

**Comment by ye-luo** [comment]
Test this please

**Review by williamfgc — APPROVED** [review]

File: tests/test_automation/github-actions/ci/run_step.sh [hunk 1]
```diff
@@ -54,8 +54,8 @@ case "$1" in
         echo "CUDACXX=/usr/local/cuda-11.2/bin/nvcc" >> $GITHUB_ENV
 
       else
-        echo "Set CUDACXX CMake environment variable to nvcc standard location"
-        export CUDACXX=/usr/local/cuda/bin/nvcc
+        echo "Set CUDACXX CMake environment variable to nvcc 11.8"
+        export CUDACXX=/usr/local/cuda-11.8/bin/nvcc
 
         # Make current environment variables available to subsequent steps
         echo "CUDACXX=/usr/local/cuda/bin/nvcc" >> $GITHUB_ENV
```
[the remark this answers is not in the document]
Comment (ye-luo) [reply tests/test_automation/github-actions/ci/run_step.sh]: I believe the previous failure was caused by missing cudatoolkit library at linking and at run and updating LD_LIBRARY_PATH fixed the issue.

**Pull request merged**